src/jira.js

```diff
@@ -16,7 +16,7 @@ export default class JiraApi {
   constructor(options) {
     this.protocol = options.protocol || 'http';
     this.host = options.host;
-    this.port = options.port || 80;
+    this.port = options.port || null;
```

Comment:
That sounds reasonable enough to change. We can still push this out as a Major version bump anyways. I've been holding off on #19 since it's also a breaking change so we can bundle the two in the next release.